projects/laji/src/app/shared-modules/select/select-subcategories/select-subcategories.component.ts

```diff
@@ -301,6 +302,8 @@ export class SelectSubcategoriesComponent implements OnChanges {
       });
     }
 
+    this.open = (!this.open && Object.keys(this.selectedOptions).length > 0) ? true : this.open;
```

Comment:
I changed the code in the commit 5c33ca9. If i go inside the method "loopInsideOptions", it means that there is some option (global or subCategory) checked. So no need to check if Object.keys(this.selectedOptions).length > 0 because it will be anyway always true. So the variable open starts in false, but if i call "loopInsideOptions" i will turn it to true.